lib/pdf-editor-view.js

```diff
@@ -56,6 +56,7 @@ export default class PdfEditorView extends ScrollView {
       }
     }, 100);
 
+    disposables.add(atom.config.onDidChange('pdf-view.nightMode', needsUpdateCallback));
```

Comment:
The `needsUpdateCallback` function triggers a full reload of the PDF (by calling `updatePdf`, which then reads the full file again and re-creates all the pages).

I'm wondering if that's needed if we just want to add or remove a CSS class from? In other words, I'm wondering if we can skip reloading the full file if the `pdf-view.nightMode` config changes and just add/remove the CSS class? 💡  

One way that might work is by introducing a new callback here instead of re-using `needsUpdateCallback` and calling that new callback instead. We'd then move the code which adds/removes the CSS class into that new callback.

What do you think? 💭 This would be a minor performance improvement in most cases, but for large PDFs it might help more than a bit since we'd skip re-creating everything.